scripts/index.js

```diff
@@ -0,0 +1,141 @@
+const featureSpeakers = [
+  {
+    id: 1,
+    name: 'Rashid Khan',
+    title: 'Focus on what you are Doing.',
+    description: 'The pain you suffer today is the sign of tomorrow’s victory.',
+    featured_image: './img/speaker_06.png',
+    alt_message: 'Featured item-one',
+  },
+  {
+    id: 2,
+    name: 'Nazoo',
+    title: 'We will help you to shape your muscles.',
+    description: 'Nothing tastes as good as being fit feels. Success never comes to those who quit.',
+    featured_image: './img/speaker_05.png',
+    alt_message: 'Featired ite-two',
+  },
+
+  {
+    id: 3,
+    name: 'Rana Ahmadi',
+    title: 'Persist Until Something Happens.',
+    description: 'Focus Until Something Happens.Power, perseverance, and discipline.',
+    featured_image: './img/speaker_03.png',
+    alt_message: 'Featured item-Three',
+  },
+  {
+    id: 4,
+    class: 'hide',
+    name: 'Gulalai Mangal',
+    title: 'Slow progress is better than no progress',
+    description: 'Push harder today if you want a different tomorrow.Where fitness comes naturally.',
+    featured_image: './img/speaker_04.png',
+    alt_message: 'featured item-four',
```

Comment:
- [ ] I would kindly you increase your `Feature Speakers` to 6 as we have it in the original design as this will show confidence in your ability to replicate any given design.